src/main/java/com/checkmarx/flow/custom/ServiceNowTracker.java

```diff
@@ -162,7 +163,9 @@ private String createServiceNowRequest(ScanRequest request) {
             throw new RuntimeException("ScanRequest object is empty");
         }
         String tag = createServiceNowTag(request);
-        return String.format("%s%s?comments=%s&sysparm_limit=", properties.getApiUrl(), INCIDENTS, tag, MAX_RECORDS);
+        String url = String.format("%s%s?sysparm_query=short_description>=%s AND commentsLIKE%s&sysparm_limit=%s", properties.getApiUrl(), INCIDENTS, request.getProduct().getProduct(), tag, MAX_RECORDS);
```

Comment:
@miguelfreitas93 as we discussed, this change is not correct for 2 reasons:

1. The `AND` clause in Service now doesn't exist, you need to use the `^` character and you cannot leave spaces between the clauses, doc here:
   * https://docs.servicenow.com/bundle/rome-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html#c_TableAPI
2.  The query won't work properly because the `comments` clause is not currently working (I also did some tests and the generated URL works on the browser but not using the current Cxflow REST call, don't know why yet) and using the `short_description` to fetch the issues using the product name.
   * NOTE: For that clause to be accurate, you should use the `STARTSWITH` clause instead of the `>=`
3. You shouldn't filter for the short description, the product string should have been added as a tag like the owner/repo/branch because that's how it's done in JIRA, the `CX` product string is added to the tags as well, that is not happening with ServiceNow, probably because this was developed some time ago, it needs to be updated.